### DISCUSSION TITLE
changing registry URL for github-tags datasource

### DISCUSSION THREAD

**Discussion by bitnik** [renovatebot/renovate#18104] [Request Help]
Hello, we have to change the registry URL for `github-tags` datasource and use a proxy. So we added following into our config

```js
  packageRules: [
    {
      matchDatasources: ["github-tags"],
      registryUrls: [
         // proxy to https://api.github.com
        "https://our.nexus.com/repository/raw-github-api-proxy/",
      ],
    }
  ]
```

Without this config added, renovate queries "https://api.github.com/repos/pre-commit/mirrors-prettier/tags?per_page=100" as expected. But when we run renovate with this config added, renovate queries "https://our.nexus.com/repository/raw-github-api-proxy/api/v3/repos/editorconfig-checker/editorconfig-checker.python/tags?per_page=100" instead of "https://our.nexus.com/repository/raw-github-api-proxy/repos/editorconfig-checker/editorconfig-checker.python/tags?per_page=100". It adds extra **/api/v3** path into the URL.

What do we do wrong? What would be the correct configuration?

**Comment by rarkins** [comment]
I'm not sure how to get it to work with the current approach. Renovate knows it's not api.github.com so falls back to treating it like GHE, which uses a /api/v3 prefix for all API calls.

A better fit would be this idea: https://github.com/renovatebot/renovate/issues/12042#issuecomment-970084019

**Reply by bitnik** [reply]
Yes, that idea would be very good! But on the other side we used the same approach for `terraform` and `terraform-provider` datasources and it just worked. So we expected that it works for github-tags as well.

**Reply by rarkins** [reply]
It might be possible to get it to work in that case. Perhaps you can look through and compare the datasources to see what they're doing different, particularly regarding the `/api/v3` part
